src/main/resources/scripts/importer/cbioportalImporter.py

```diff
@@ -520,18 +525,25 @@ def main(args):
     module_logger.addHandler(error_handler)
     LOGGER = module_logger
 
-    # jar_path is optional. If not set, try to find it relative to this script
-    if args.jar_path is None:
+    # java_opts is optional. If class (jar) path is not set (-cp), try to find the jar path relative to this script
+    locate_jar_path = True
+    if args.java_opts is not None and '-cp' in args.java_opts:
+        locate_jar_path = False
+    if locate_jar_path:
         try:
-            args.jar_path = locate_jar()
+            jar_path = locate_jar()
         except FileNotFoundError as e:
             print(e)
             sys.exit(2)
-        print('Data loading step using', args.jar_path)
+        print('Data loading step using', jar_path)
         print()
+        if args.java_opts is None:
+            args.java_opts = f"-cp {jar_path}"
+        else:
+            args.java_opts = f"-cp {jar_path} {args.java_opts}"
```

Comment:
Will this not become `-cp -cp JAR_PATH OTHER_JAVA_OPTS` if -cp is specified in java_opts?